x/posts/keeper/external_hooks_test.go

```diff
@@ -149,8 +117,7 @@ func (suite *KeeperTestsuite) TestKeeper_AfterSubspaceDeleted() {
 				tc.store(ctx)
 			}
 
-			// Call the method that should call the hook
-			suite.sk.DeleteSubspace(ctx, tc.subspaceID)
+			suite.k.Hooks().AfterSubspaceDeleted(ctx, tc.subspaceID)
```

Comment:
`AfterSubspaceDeleted` method should be tested by using itself instead of calling `x/subspaces` method. Testing if the hooks are called or not should be in the `x/subspaces`.

Reply:
Since we are removing this, are we sure the hooks are called properly? Are we checking that?

Reply:
@RiccardoM It is not checked inside x/subspaces, added the tests for it.